bin/fetchdata.js

```diff
@@ -15,6 +15,18 @@ axiosRetry(axios, {
     retryDelay: () => (Math.random() * 2000) + 1000      // jitter retry delay by 1 - 3 seconds
 });
 
+function populateMissingData(attributes, id) {
+    if (id === '3941784506' || id === '3463348370') {
```

Comment:
Few suggestions:

1. Let's add comments for the names here
2. Can we change to a switch statement. TS supports stacked cases so you can do:

```typescript
switch (id) {
    case '3941784506':  // clone trooper
    case '3463348370':  // battle droid
        // ...code here...
```

I just guessed at those IDs, please confirm if they're right or not 😂